src/Kiota.Builder/PluginAuthType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Kiota.Builder;
+
+public enum PluginAuthType
```

Comment:
this will be confusing with the one in #5209
We should find better names for both since I believe they represent different aspects. (one is the OpenAPIAuthType)